pkg/stanza/docs/operators/syslog_parser.md

```diff
@@ -14,6 +14,7 @@ The `syslog_parser` operator parses the string-type field selected by `parse_fro
 | `protocol`                           | required         | The protocol to parse the syslog messages as. Options are `rfc3164` and `rfc5424`. |
 | `location`                           | `UTC`            | The geographic location (timezone) to use when parsing the timestamp (Syslog RFC 3164 only). The available locations depend on the local IANA Time Zone database. [This page](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) contains many examples, such as `America/New_York`. |
 | `enable_octet_counting`              | `false`          | Wether or not to enable [RFC 6587](https://www.rfc-editor.org/rfc/rfc6587#section-3.4.1) Octet Counting on syslog parsing (Syslog RFC 5424 only).  |
+| `allow_skip_pri_header`              | `false`          | Allow parsing records without the pri header. If the header is missing the message will be parsed and the priority and facility field will be set to their zero values. By default records without pri header will throw an exception. |
```

Comment:
We should add this configuration option to Syslog receiver docs as well.

Reply:
by Syslog receiver do you mean syslog_input.md? because I already added it to receiver/syslogreceiver/README.md

Reply:
Oh sorry, I don't know how I missed it. You have indeed added this to the Syslog receiver's README.

Reply:
Right, this is already added to the Syslog receiver's README. I don't know how I missed this, sorry!